### ISSUE TITLE
App goes slow with like 200 images

### ISSUE THREAD

**Issue by Ruzen1** [DylanVann/react-native-fast-image#816]
**Describe the bug**
I have a grid of images. It works fine till 169 of them. But when there are more that that , the app turns slow.

**Expected behavior**
It should not turn slow. It should work smmot

**Dependency versions**

- React Native version: 0.61.3
- React version: 16.9.0
- React Native Fast Image version: 7.0.2

Unfortunately I cannot upgrade the versions.

**Comment by mhdSid** [comment]
I have been trying to make my app fast on iPhone 6. The app uses FastImage. After trying so many workarounds and debugging FastImage code, I have decided to use React Native Image element and react native image resizer. If your app displays local images, you can compress them to Jpg and then you will notice that difference in performance. If your app uploads images, you can compress them using the library before uploading them. This solution works well because the native Image component loves jpeg.

**Comment by aleluiah** [comment]
> 

i think i will agree with you

**Comment by LostAnurag** [comment]
> I have been trying to make my app fast on iPhone 6. The app uses FastImage. After trying so many workarounds and debugging FastImage code, I have decided to use React Native Image element and react native image resizer. If your app displays local images, you can compress them to Jpg and then you will notice that difference in performance. If your app uploads images, you can compress them using the library before uploading them. This solution works well because the native Image component loves jpeg.

yes using jpeg image will definitely improve the app performance in react native image
